include/jitify_options.hpp.in

```diff
@@ -1,3 +1,6 @@
-#define JITIFY_OPTIONS -I${CMAKE_BINARY_DIR}/lib \
--I${CMAKE_BINARY_DIR}/include \
--I${CMAKE_BINARY_DIR}/include/externals
+#define JITIFY_OPTIONS -I${CMAKE_BINARY_DIR}/lib        \
```

Comment:
Should that whole file move to include/targets/cuda ?

Reply:
Good point.  I've moved the jitify headers to `include/targets/cuda` and done a bit of extra cleanup (4c330f583).